lib/model/migrations/20171010-01-auth.js

```diff
@@ -0,0 +1,79 @@
+
+const up = (knex, Promise) => {
+  const createActees = knex.schema.createTable('actees', (actees) => {
+    actees.string('id', 36).primary();
+    actees.string('species', 36);
```

Comment:
This is not an enum so that it can also be a reference to a specific model?

Reply:
So it can be a reference to a species. So for instance the default admin group we'd inject would be granted all permissions on all users, without a grant being created for every single user, as well as to describe the `create` verb for a species.